index.js

```diff
@@ -4,18 +4,21 @@ let Promise = require('bluebird')
 let readDir = Promise.promisify(require('recursive-readdir'))
 let fs = Promise.promisifyAll(require('fs'))
 
+let slice = start => thing => thing.slice(start)
+
 // Path Utils
-let filesRelative = (dir, ex) => readDir(dir, ex).map(x => x.slice(dir.length))
+let relativeFilenames = (dir, exclusions) => readDir(dir, exclusions).map(n => slice(dir.length)(n))
```

Comment:
The point of pulling this out was to be point free - there's no need to declare a function in map since slice is curried. Just do something like this:
```js
let relativeFilenames = (dir, exclusions) => readDir(dir, exclusions).map(slice(dir.length))
```